lock.sbt

```diff
@@ -5,46 +5,31 @@ dependencyOverrides in ThisBuild ++= Seq(
   "ch.qos.logback" % "logback-classic" % "1.2.3",
   "ch.qos.logback" % "logback-core" % "1.2.3",
   "com.chuusai" % "shapeless_2.12" % "2.3.2",
-  "com.fasterxml.jackson.core" % "jackson-annotations" % "2.8.9",
-  "com.fasterxml.jackson.core" % "jackson-core" % "2.8.9",
-  "com.fasterxml.jackson.core" % "jackson-databind" % "2.8.9",
-  "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.8.9",
-  "com.fasterxml.jackson.module" % "jackson-module-paranamer" % "2.8.9",
-  "com.fasterxml.jackson.module" % "jackson-module-scala_2.12" % "2.8.9",
   "com.github.mpilquist" % "simulacrum_2.12" % "0.10.0",
-  "com.github.swagger-akka-http" % "swagger-akka-http_2.12" % "0.10.0",
-  "com.google.guava" % "guava" % "20.0",
+  "com.google.guava" % "guava" % "19.0",
```

Comment:
Why go down from 20.0 to 19.0?

Reply:
Library, that required guava 20 was removed, we now only have libraries that require guava 19 in dependencies